utilities/dumb_qualisys_joint_center_dictionary_builder.py

```diff
@@ -0,0 +1,37 @@
+
+def dumb_qualisys_joint_center_dictionary_builder() -> dict:
+
+    # this is a special purpose dictionary for getting ARGP skeleton data from qualisys
+    # into a generic meadiapipe++ format
+
+    # below is a handwritten dictionary, where the key represents the joint center name
+    # that is going into the skeleton, the first value represents the physical qualisys
+    # markers that are being combined to be put into the joint center, and the second value
+    # (or array of values) is the relative weight of each of the markers for the joint
+
+    qualisys_joint_center_dict = {
+        "head_xyz": [["HeadL+HeadR"], [0.5, 0.5]],
+        "left_head_marker_xyz": [["HeadL"], [1]],
```

Comment:
Formatting is kinda wrong - You're making a 'dictionary full of kinda janky lists' when you should be making a 'dictionary of dictionaries'

Reply:
formatting is fixed!